ToDoList-Redux/Binders/ReduxFramework.swift

```diff
@@ -14,54 +14,72 @@ import CombineRex
 // MARK: - ACTIONS
 
 enum AppAction {
-    case task(TaskList.Action)
+    case appLifecycle(AppLifecycleAction)
+    case task(TaskAction)
 }
 
+enum TaskAction {
+    case add(String)
+    case remove(IndexSet)
+    case move(IndexSet, Int)
+    case toggle(String)
```

Comment:
In the "model" level of toggle action we still need the id. But we remove the responsibility of keeping track of this id from the view, making it more "dumb".